README.md

```diff
@@ -1,224 +1,118 @@
-  [capybara_accessible_selectors](https://github.com/citizensadvice/capybara_accessible_selectors)
-* [Factory Bot](https://github.com/thoughtbot/factory_bot) for test data
-* [Formulaic](https://github.com/thoughtbot/formulaic) for integration testing
-  HTML forms
-* [RSpec](https://github.com/rspec/rspec) for unit testing
-* [RSpec Mocks](https://github.com/rspec/rspec-mocks) for stubbing and spying
-* [Shoulda Matchers](https://github.com/thoughtbot/shoulda-matchers) for common
-  RSpec matchers
+It is used by thoughtbot to get a jump start on a new or existing app. Use
+Suspenders if you're in a rush to build something amazing; don't use it if you
+like missing deadlines.
 
-## Other goodies
+[Rails Engine]: https://guides.rubyonrails.org/engines.html
+[features]: ./FEATURES.md
 
-Suspenders also comes with:
+![Suspenders boy](https://media.tumblr.com/1TEAMALpseh5xzf0Jt6bcwSMo1_400.png)
 
-* The [`./bin/setup`][setup] convention for new developer setup
-* The `./bin/deploy` convention for deploying to Heroku
-* Rails' flashes set up and in application layout
-* A few nice time formats set up for localization
-* `Rack::Deflater` to [compress responses with Gzip][compress]
-* A [low database connection pool limit][pool]
-* [Safe binstubs][binstub]
-* [t() and l() in specs without prefixing with I18n][i18n]
-* An automatically-created `SECRET_KEY_BASE` environment variable in all
-  environments
-* Configuration for [CircleCI][circle] Continuous Integration (tests)
-* Configuration for [Hound][hound] Continuous Integration (style)
-* Configuration for [stylelint][stylelint]
-* The analytics adapter [Segment][segment] (and therefore config for Google
-  Analytics, Intercom, Facebook Ads, Twitter Ads, etc.)
-* [PostCSS Autoprefixer][autoprefixer] for CSS vendor prefixes
-* [PostCSS Normalize][normalize] for resetting browser styles
+## Usage
 
-[setup]: https://robots.thoughtbot.com/bin-setup
-[compress]: https://robots.thoughtbot.com/content-compression-with-rack-deflater
-[pool]: https://devcenter.heroku.com/articles/concurrency-and-database-connections
-[binstub]: https://github.com/thoughtbot/suspenders/pull/282
-[i18n]: https://github.com/thoughtbot/suspenders/pull/304
-[circle]: https://circleci.com/docs
-[hound]: https://houndci.com
-[stylelint]: https://stylelint.io/
-[segment]: https://segment.com
-[autoprefixer]: https://github.com/postcss/autoprefixer
-[normalize]: https://github.com/csstools/postcss-normalize
+Suspenders can be used to create a new Rails application, or to enhance an
+existing Rails application.
 
-## Heroku
+### With New Rails Applications
 
-Read the documentation on [deploying to Heroku][heroku deploy]
+This approach uses an [application template][] to generate a new Rails
+application with Suspenders.
 
-You can optionally create Heroku staging and production apps:
+We skip the [default test framework][] in favor of [RSpec][], and [prefer
+PostgreSQL][] as our database.
```

Comment:
Once https://github.com/thoughtbot/dotfiles/pull/744 is merged, and once we add the template to the `railsrc` file, we might want to link to our dotfiles, or at least highlight that a `railsrc` file can help simplify this.